x-pack/plugins/security_solution/public/management/pages/policy/view/policy_forms/protections/ransomware.tsx

```diff
@@ -0,0 +1,343 @@
+                        defaultMessage="
+                    The user notification can be customized in the text box below. Bracketed tags can be used to dynamically populate the applicable action (such as prevented or detected) and the filename."
+                      />
+                    </>
+                  }
+                />
+              </EuiFlexItem>
+            </EuiFlexGroup>
+            <EuiSpacer size="xs" />
+            <EuiTextArea
+              placeholder={i18n.translate(
+                'xpack.securitySolution.endpoint.policyDetails.ransomware.userNotification.placeholder',
+                {
+                  defaultMessage: 'Input your custom notification message',
+                }
+              )}
+              value={userNotificationMessage}
+              onChange={handleCustomUserNotification}
+              fullWidth={true}
+              data-test-subj="ransomwareUserNotificationCustomMessage"
+            />
+          </>
+        )}
+      </>
+    );
+  }, [
+    radios,
+    selected,
+    handleUserNotificationCheckbox,
+    userNotificationSelected,
+    userNotificationMessage,
+    handleCustomUserNotification,
+  ]);
+
+  const protectionSwitch = useMemo(() => {
+    return (
+      <EuiSwitch
+        label={i18n.translate(
+          'xpack.securitySolution.endpoint.policy.details.ransomwareProtectionsEnabled',
+          {
+            defaultMessage:
+              'Ransomware protections {mode, select, true {enabled} false {disabled}}',
+            values: {
+              mode: selected !== ProtectionModes.off,
+            },
+          }
+        )}
+        checked={selected !== ProtectionModes.off}
+        onChange={handleSwitchChange}
+      />
+    );
+  }, [handleSwitchChange, selected]);
+
+  return (
+    <ConfigForm
+      type={i18n.translate('xpack.securitySolution.endpoint.policy.details.ransomware', {
+        defaultMessage: 'Ransomware',
+      })}
+      supportedOss={[OperatingSystem.WINDOWS, OperatingSystem.MAC]}
+      dataTestSubj="ransomwareProtectionsForm"
+      rightCorner={protectionSwitch}
+    >
+      {radioButtons}
+      <EuiSpacer size="m" />
+      <EuiCallOut iconType="iInCircle">
+        <FormattedMessage
+          id="xpack.securitySolution.endpoint.policy.details.detectionRulesMessage"
+          defaultMessage="View {detectionRulesLink}. Prebuilt rules are tagged “Elastic” on the Detection Rules page."
+          values={{
+            detectionRulesLink: (
+              <LinkToApp appId={`${APP_ID}:${SecurityPageName.detections}`} appPath={`/rules`}>
+                <FormattedMessage
+                  id="xpack.securitySolution.endpoint.policy.details.detectionRulesLink"
+                  defaultMessage="related detection rules"
+                />
+              </LinkToApp>
+            ),
+          }}
+        />
+      </EuiCallOut>
+    </ConfigForm>
+  );
+});
+
+Ransomware.displayName = 'RansomwareProtections';
```

Comment:
Are there significant differences between ransomware and malware sections?

Reply:
mainly its just what is considered a paid feature. malware considers the user notification portion of the card a paid feature, whereas ransomware considers the entire card itself a paid feature

Reply:
Does it make things harder to share between the two forms?

Reply:
mmm maybe, i think its just that i don't foresee any other features having half the card be restricted by license. i would say if there is another card like ransomware where the whole card only for platinum license users, than it makes sense to make a more reusable component